brightwind/export/export.py

```diff
@@ -154,3 +154,97 @@ def export_csv(data, file_name=None, folder_path=None, **kwargs):
     else:
         raise NotADirectoryError("The destination folder doesn't seem to exist.")
     print('Export to csv successful.')
+
+
+def export_tws_file(eastings, northings, height, wspd_series, direction_series, wspd_std_series=None, site_name=None, file_name=None, folder_path=None):
+    """Export a WindSim timeseries tws file using a timeseries dataframe of wind speed and direction.
```

Comment:
The description mentions 'timeseries dataframe of wind speed...' but you input a series, suggest removing 'dataframe'.

Reply:
Updated.